src/components/study-container.js

```diff
@@ -242,31 +245,38 @@ export function StudyContainer({ view, onChangeTab }) {
 
     const wsRef = useRef();
 
-    const checkFailNotifications = useCallback(
+    const displayErrorNotifications = useCallback(
         (eventData) => {
             const updateTypeHeader = eventData.headers[UPDATE_TYPE_HEADER];
+            const errorMessage = eventData.headers[ERROR_HEADER];
+            const userId = eventData.headers[USER_HEADER];
+            if (userId !== userName) return;
```

Comment:
Perhaps test if `eventData.headers[USER_HEADER] ` is not null ?

Reply:
Hmm why, in this case it will just satisfy the condition and return directly no ?

Reply:
ok